### PR TITLE
[zh] 2020-05-06-third-party-dual-sourced-content.md

### PR DESCRIPTION
**Pull request by lis102**
translation

### CODE REVIEW

File: content/en/blog/_posts/2020-05-06-third-party-dual-sourced-content.md [hunk 1]
```diff
@@ -1,60 +1,143 @@
...
 *Editor's note: Zach is one of the chairs for the Kubernetes documentation special interest group (SIG Docs).*
+-->
+**作者：** Zach Corleissen，云原生计算基金会
 
+*编者注： Zach 是 Kubernetes 文档特别兴趣小组（SIG Docs）的主席之一。*
+
+<!--
 Late last summer, SIG Docs started a community conversation about third party content in Kubernetes docs. This conversation became a [Kubernetes Enhancement Proposal](https://github.com/kubernetes/enhancements/pull/1327) (KEP) and, after five months for review and comment, SIG Architecture approved the KEP as a [content guide](/docs/contribute/style/content-guide/) for Kubernetes docs.
+-->
+去年夏末， SIG Docs 就 Kubernetes 文档中的第三方内容展开了一次社区会谈。这次讨论构建了 [Kubernetes Enhancement Proposal](https://github.com/kubernetes/enhancements/pull/1327)(KEP)，经过五个月的审查和评议， SIG Architecture 批准了 KEP 作为 Kubernetes 文档的 [内容指南](/docs/contribute/style/content-guide/)。
```
Comment (tengqm) [inline content/en/blog/_posts/2020-05-06-third-party-dual-sourced-content.md:26]: 构建不合适
内容指南前无需空格
链接应指向中文版本

File: content/en/blog/_posts/2020-05-06-third-party-dual-sourced-content.md [hunk 2]
```diff
@@ -1,60 +1,143 @@
...
+这是现在 Kubernetes 文档怎样处理第三方内容的：
 
+<!--
 > Links to active content in the Kubernetes project (projects in the kubernetes and kubernetes-sigs GitHub orgs) are always allowed.
 >
 > Kubernetes requires some third party content to function. Examples include container runtimes (containerd, CRI-O, Docker), networking policy (CNI plugins), Ingress controllers, and logging.
 >
 > Docs can link to third party open source software (OSS) outside the Kubernetes project if it’s necessary for Kubernetes to function.
+-->
+>总是允许链接到 Kubernetes 项目（kubernetes 和 kubernetes-sigs GitHub orgs 中的项目）中的活动内容。
```
Comment (tengqm) [inline content/en/blog/_posts/2020-05-06-third-party-dual-sourced-content.md:40]: '>' 之后需要一个空格，这是 markdown 的语法

File: content/en/blog/_posts/2020-05-06-third-party-dual-sourced-content.md [hunk 3]
```diff
@@ -1,60 +1,143 @@
...
+>
+>Kubernetes 需要一些第三方内容发挥作用。包括容器运行（容器， CRI-O， Docker），网络策略（ CNI 插件），入口控制器和日志等示例。
```
Comment (tengqm) [inline content/en/blog/_posts/2020-05-06-third-party-dual-sourced-content.md:42]: 有些空格莫名奇妙

File: content/en/blog/_posts/2020-05-06-third-party-dual-sourced-content.md [hunk 4]
```diff
@@ -1,60 +1,143 @@
...
 Over time, SIG Docs observed increasing vendor content in the docs. Some content took the form of vendor-specific implementations that aren't required for Kubernetes to function in-project. Other content was thinly-disguised advertising with minimal to no feature content. Some vendor content was new; other content had been in the docs for years. It became clear that the docs needed clear, well-bounded guidelines for what kind of third party content is and isn't allowed. The [content guide](https://kubernetes.io/docs/contribute/content-guide/) emerged from an extensive period for review and comment from the community.
+-->
+## 背景
+
+随着时间的推移， SIG Docs 发现文档中供应商的内容正在增加。一些内容采用了特定于供应商的形式，而 Kubernetes 在项目中运行并不需要。另一些是伪装的广告，内容很少甚至没有。一些供应商的内容是新的，另一些内容已经存在于文档中多年。显然，文档需要清晰、明确的准则，以指示允许和不允许的第三方内容类型。 [内容指南](https://kubernetes.io/docs/contribute/content-guide/)已经在社区中评议和评论了很长时间。
 
+<!--
 Docs work best when they're accurate, helpful, trustworthy, and remain focused on features. In our experience, vendor content dilutes trust and accuracy. 
+-->
+当文档准确，有用，值得信赖且始终专注于功能时，它们才能发挥最佳性能。根据我们的经验，供应商的内容会削弱信任度和准确性。
```
Comment (tengqm) [inline content/en/blog/_posts/2020-05-06-third-party-dual-sourced-content.md:88]:
```suggestion
当文档准确、有用、值得信赖且始终专注于功能时，它们才能发挥最好的作用。根据我们的经验，供应商的内容会削弱信任度和准确性。
```

File: content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+### Re-homing content
+-->
+### 重新定位内容
+
+<!--
+Some content will be removed that readers may find helpful. To make sure readers have continous access to information, we're giving stakeholders until the [1.19 release deadline for docs](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.19), **July 9th, 2020** to re-home any content slated for removal.
+-->
+某些读者认为有帮助的内容将被删除。为了确保读者可以持续访问这些信息，我们将向相关者保持开放直至[1.19文档发布截止日期](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.19)。**2020年7月9日**会将原定要删除的所有内容重新存放。
```
Comment (howieyuen) [inline content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md:69]: 1.19 文档发布截止日期
数字前后分别加一个空格

File: content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+Over the next few months you'll see less third party content in the docs as contributors open PRs to remove content.
+-->
+在接下来的几个月中，随着贡献者打开 PR 删除内容，您将在文档中看到更少的第三方内容。
```
Comment (howieyuen) [inline content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md:74]: 您 -> 你

File: content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+## Dual sourced content
+-->
+## 双重来源的内容
+
+<!--
+Less impactful but also important is how Kubernetes docs handle _dual-sourced content_. Dual-sourced content is content published in more than one location, or from a non-canonical source.
+-->
+Kubernetes 文档如何处理_dual-sourced content_虽然影响较小但同样重要。双源内容是在多处或从非规范来源发布的内容。
```
Comment (howieyuen) [inline content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md:103]: _dual-sourced content_ 前后加个空格

File: content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+## Ways to contribute
+-->
+## 贡献的方式
+
+<!--
+We're tracking third-party content in an [issue in the Kubernetes website repository](https://github.com/kubernetes/website/issues/20232). If you see third party content that's out of project and isn't required for Kubernetes to function, please comment on the tracking issue. 
+-->
+我们正在 [issue in the Kubernetes website repository](https://github.com/kubernetes/website/issues/20232) 跟踪第三方内容。如果您看到第三方内容超出项目范围并且不是 Kubernetes 正常运行所必需的，请对跟踪问题发表评论。
```
Comment (howieyuen) [inline content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md:128]: Kubernetes website 仓库的 issue 中

File: content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md [hunk 9]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+Feel free to open a PR that removes non-conforming content once you've identified it!
+-->
+识别出不合格内容后，即可随意打开PR！
```
Comment (howieyuen) [inline content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md:133]: 空格
即可随意打开 PR！

File: content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md [hunk 10]
```diff
@@ -0,0 +1,143 @@
...
+
+<!--
+## Want to know more?
+-->
+## 想了解更多？
+
+<!--
+For more information, read the issue description for [tracking third party content](https://github.com/kubernetes/website/issues/20232). 
+-->
+更多信息，请阅读问题描述[tracking third party content](https://github.com/kubernetes/website/issues/20232)。
```
Comment (howieyuen) [inline content/zh/blog/_posts/2020-05-06-third-party-dual-sourced-content.md:143]: 追踪第三方内容

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/en/blog/_posts/2020-05-06-third-party-dual-sourced-content.md [hunk 11]
```diff
@@ -57,4 +57,4 @@ Feel free to open a PR that removes non-conforming content once you've identifie
 
 ## Want to know more?
 
-For more information, read the issue description for [tracking third party content](https://github.com/kubernetes/website/issues/20232). 
+For more information, read the issue description for [tracking third party content](https://github.com/kubernetes/website/issues/20232).
```
Comment (onlydole) [inline content/en/blog/_posts/2020-05-06-third-party-dual-sourced-content.md:60]:
```suggestion
For more information, read the issue description for [tracking third party content](https://github.com/kubernetes/website/issues/20232).
```
It'd be helpful to remove these extra spaces

**Review by onlydole — CHANGES_REQUESTED** [review]
LGTM with one nit

**Comment by tengqm** [comment]
@lis102 are you still working on this?

**Comment by tengqm** [comment]
@lis102 Please respond to the review comments when convenient. Thanks.

**Comment by kbhawkey** [comment]
👋 @lis102 .
Are you working on this PR? Reach out if you need help.

**Comment by tengqm** [comment]
Feel free to reopen this when you get cycle to continue.
/close

**Pull request closed without merging**